src/MIDebugEngine/Engine.Impl/ExceptionManager.cs

```diff
@@ -311,6 +312,17 @@ public bool TryGetExceptionBreakpoint(string bkptno, out string exceptionName, o
                         {
                             exceptionName = categorySettings.CategoryName;
                         }
+
+                        string functionName = frame?.TryFindString("func");
+                        if (string.IsNullOrWhiteSpace(functionName))
+                        {
+                            exceptionDescription = string.Format(CultureInfo.CurrentCulture, ResourceStrings.Exception_Thrown, address);
```

Comment:
>Exception_Thrown [](http://example.com/codeflow?start=109&length=16)

Does this string show up in VS? If so, should we include the name of the exception that we are stopping on when this isn't a category exception?